CLI mistake in aliases for compare-states

```
compare-states flag redefined: h
panic: compare-states flag redefined: h

goroutine 1 [running]:

```

should be :
```
Name:    "ignore-height",
Aliases: []string{"g"},
```

--- a/scripts/compare-states/compare-states.go
+++ b/scripts/compare-states/compare-states.go
@@ -169,7 +169,7 @@ func main() {
 	ctl.Flags = []cli.Flag{
 		&cli.BoolFlag{
 			Name:    "ignore-height",
-			Aliases: []string{"h"},
+			Aliases: []string{"g"},
 			Usage:   "Ignore height difference",
 		},
 	}

